sovrin_client/agent/agent_prover.py

```diff
@@ -69,24 +69,32 @@ def sendReqClaim(self, link: Link, schemaKey):
 
         self.signAndSend(msg=op, linkName=link.name)
 
-    def sendProofReq(self, link: Link, schemaKey):
-        if self.loop.is_running():
-            self.loop.call_soon(asyncio.ensure_future,
-                                self.sendProofReqAsync(link, schemaKey))
+    def handleProofRequest(self, msg):
+        body, _ = msg
+        link = self._getLinkByTarget(getCryptonym(body.get(IDENTIFIER)))
+        proofReqName = body.get(NAME)
```

Comment:
Sending a proof request is now updated to send with version. However, since we don't show version in proof request as of now, I have not updated prover side to along with version. I will create a story in jira to track the same.